dom-event-delegation/main.js

```diff
@@ -0,0 +1,13 @@
+var taskList = document.querySelector('.task-list');
+var taskListItem = document.querySelector('.task-list-item');
+
+taskList.addEventListener('click', taskListCallback);
+function taskListCallback(e) {
+  console.log('event target:', e.target);
+  console.log('event tagname:', e.target.tagName);
+  if (e.target && e.target.nodeName === 'BUTTON') {
+
+    console.log(taskList.closest('.task-list-item'));
+    taskListItem.remove();
```

Comment:
You don't want to remove `taskListItem`, you want to remove the `event.target.closest('.task-list-item')`

Reply:
I was looking at that and I understand it now.  Will go back and review it also.  Thanks!

Reply:
![56456456476](https://user-images.githubusercontent.com/122063836/217987681-d44ef6ab-4758-440f-867a-a086766f87bd.gif)